contracts/serverless-v2/Jobs.sol

```diff
@@ -271,7 +273,7 @@ contract Jobs is
         bytes32 digest = keccak256(abi.encodePacked("\x19\x01", DOMAIN_SEPARATOR, hashStruct));
         address signer = digest.recover(_signature);
 
-        executors.allowOnlyVerified(signer);
+        executors.allowOnlyVerified(signer, operator);
 
         if(!isJobExecutor(_jobId, signer))
```

Comment:
shouldn't it be operator instead of signer. Because selectedExecutor is now using operators in the list.

Reply:
yes, fixed this